content/configuration/tasks-build.md

```diff
@@ -28,6 +28,10 @@ You can obviously use any other image name as long as it contains the Docker reg
 
 ContinuousPipe is getting parameters from your `docker-compose.yml` file. The `image` is the only required value if you have such a Docker Compose file.
 
+{{< warning title="Warning" >}}
+Always think carefully before pushing your images to a public Docker repository. If you are using a private Git repository or your build contains secrets (e.g. Github token, database passwords) then you need to ensure that you use a private Docker repository such as [Quay](https://quay.io/).
```

Comment:
Being on quay doesn't automatically make a repo private, though the default is to be private. Docker Hub also supports private images. Both options are paid though.